src/features/island/hud/components/deliveries/TravelModal.tsx

```diff
@@ -32,44 +48,74 @@ export const TravelModal: React.FC<Props> = ({ isOpen, onClose }) => {
         onShow={() => gameService.send("SAVE")}
         dialogClassName="md:max-w-3xl"
       >
-        <CloseButtonPanel
-          onClose={onClose}
-          tabs={[
-            { icon: world, name: "Travel" },
-            { icon: SUNNYSIDE.icons.heart, name: "Deliveries" },
-            { icon: SUNNYSIDE.icons.expression_chat, name: "Chores" },
-          ]}
-          currentTab={tab}
-          setCurrentTab={setTab}
-        >
-          <div
-            style={{ maxHeight: CONTENT_HEIGHT }}
-            className="w-full pr-1 pt-2.5 overflow-y-auto scrollable"
+        {showIntro ? (
+          <Panel bumpkinParts={NPC_WEARABLES["reelin roy"]}>
```

Comment:
Any thoughts on some Sailor NPC we could use on this modal?

Do you think there are any wearables which would give the impression of a 'traveller' or 'explorer'